examples/chart/teleport-cluster/charts/teleport-operator/Chart.yaml

```
@@ -4,7 +4,7 @@ name: teleport-operator
 apiVersion: v2
 version: *version
 appVersion: *version
-description: Teleport Operator is a unified access plane for your infrastructure
+description: Teleport Operator provides management of select Teleport resources following the Operator Pattern.
```

```suggestion
description: Teleport Operator provides management of select Teleport resources.
```

Up to you, but it kind of sounded like we were saying "Teleport operator is an operator for Teleport"